modules/core-components/src/main/java/org/mule/runtime/core/internal/routing/IdempotentMessageValidator.java

```diff
@@ -169,12 +178,21 @@ private boolean accept(CoreEvent event) {
           store.store(id, value);
           return true;
         } catch (ObjectAlreadyExistsException ex) {
+          if (featureFlaggingService.isEnabled(RETHROW_EXCEPTIONS_IN_IDEMPOTENT_MESSAGE_VALIDATOR)) {
```

Comment:
this is the actual duplicatew error, we want to keep the original behaviour here